build/Linux+BSD/portable/copy-libs

```diff
@@ -0,0 +1,176 @@
+#!/bin/sh
+
+# Usage:   copy-libs  $share_path
+
+# This file is designed to run on CentOS 6.7 but may work on other systems.
+
+main() {
+  # Set sources:
+  qt_prefix="$(locateQt)"
+  echo "qt_prefix: $qt_prefix"
+
+  # Set destinations:
+  lib_dest="$1/lib"
+  qt_dest="$lib_dest/qt5"
+  echo "lib_dest: $lib_dest"
+  echo "qt_dest: $qt_dest"
+  
+  getCrossPlatformDependencies
+  getLinuxOnlyDependencies
+}
+
+##########################################################################
+# LINUX EQUIVALENTS OF WINDOWS DEPENDENCIES (see "mscore/CMakeLists.txt")
+##########################################################################
+getCrossPlatformDependencies() {
+  # Non-Qt dependencies (see "mscore/CMakeLists.txt")
+  dest_dir="$lib_dest"
+  #copyLib libgcc_s_dw2-1.so # Assume provided by system
+  #copyLib libstdc++-6.so # Assume provided by system
+  #copyLib libwinpthread-1.so # Assume provided by system
+  #copyLib libogg.so # Assume provided by system
+  #copyLib libsndfile-1.so # Assume provided by system
```

Comment:
I tested on Ubuntu, Fedora, openSUSE and CentOS and these were all available. I think distros are pretty good about including codecs (except non-free ones like MP3) so that media will "just work" out of the box.

I included a new command line options "check-depends" which will scan the system for included libraries. If someone has a problem with the AppImage we can ask them to do:
<code>MuseScore*.AppImage check-depends > dependencies.txt</code>
Then can upload the "dependencies.txt" file to the forum and we will know what they are missing.